specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2019-05-01/NasCluster.json

```diff
@@ -0,0 +1,207 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Nas Cluster operation endpoints and objects.",
+    "title": "FabricAdminClient",
+    "version": "2019-05-01"
+  },
+  "host": "adminmanagement.local.azurestack.external",
```

Comment:
just want to confirm. This is by design. Right?

Reply:
OK. change the value to 'management.azure.com' to align with other spec

Reply:
well. This is an important field. It decides the endpoint which the rest call will be sent. It doesn't matter whether it's consistent with other spec or not. Please double confirm it.

Reply:
This would not go to Azure as it is azure stack Hub API. But the build check enforce the host to be "management.azure.com"

Reply:
pls change it to management.azure.com, it is not used anywhere

Reply:
Already updated